src/aws-healthomics-mcp-server/README.md

```diff
@@ -184,7 +184,7 @@ Or install from source:
 git clone <repository-url>
 cd mcp/src/aws-healthomics-mcp-server
 uv sync
-uv run server.py
+python -m awslabs.aws_healthomics_mcp_server.server
```

Comment:
Thanks for spotting this. Keeping with using `uv` the command `uv run -m awslabs.aws_healthomics_mcp_server.server` would be preferred over `python`. Let me know if that works in your environment.